libs/payments/metrics/src/lib/glean/glean.manager.spec.ts

```diff
@@ -156,4 +164,70 @@ describe('PaymentsGleanManager', () => {
       });
     });
   });
+
+  describe('enabled is false', () => {
+    {
+      let paymentsGleanManager: PaymentsGleanManager;
+      let paymentsGleanServerEventsLogger: any;
+
+      beforeEach(async () => {
```

Comment:
This doesn't look like it has an resetting logic, can it be changed to `beforeAll` for test performance?

Reply:
If this winds up being flakey, it may be worthwhile having an `afterAll` method which resets the instance as well. I'm not sure how hoisting effects the order of operations here.

Reply:
Ah sweet, yup good spot, thank you. I made the change.